### PR TITLE
Document the cdiLiteMode flag and add a core profile config example, …

### PR DESCRIPTION
**Pull request by starksm64**
…#335

Signed-off-by: Scott M Stark <starksm64@gmail.com>

### CODE REVIEW

File: doc/reference/src/main/asciidoc/configuration.asciidoc [hunk 1]
```diff
@@ -17,8 +17,7 @@ You should set the following required properties:
 [options="header"]
 |===============
 |Property = Example Value|Description
```
Comment (Ladicek) [inline doc/reference/src/main/asciidoc/configuration.asciidoc:19]: Maybe this table (or the one below?) should also have a line for the `cdiLiteMode` flag?

**Comment by Ladicek** [comment]
Other than that, looks fine.

**Review by Ladicek — APPROVED** [review]

**Comment by manovotn** [comment]
So what's the purpose of this property? Just to indicate that other configuration is not needed?

**Comment by starksm64** [comment]
That is what it has become at this point. In previous iterations of the build time stuff we were also adding source, and triggering the annotation processor.

**Comment by manovotn** [comment]
I knew we were. I am rather asking we if still need this configuration option for something.

**Comment by starksm64** [comment]
It is annoying to have to set the various jdbc, jms properties if you are running under CDI Lite. Do you have an alternative suggestion for how to deal with this?

**Comment by manovotn** [comment]
> It is annoying to have to set the various jdbc, jms properties if you are running under CDI Lite

Aah, so it is there to avoid the other configs. Ok, got it now.
Let's keep it, I just wasn't sure I understood it completely. Sorry for the confusion.

**Pull request merged**